packages/@aws-cdk/aws-apigateway/test/integ.base-path-mapping.ts

```diff
@@ -24,7 +24,7 @@ export class TestStack extends cdk.Stack {
     new apigateway.BasePathMapping(this, 'MappingTwo', {
       domainName,
       restApi,
-      basePath: 'path',
+      basePath: 'api/v1/multi-level-path',
       attachToStage: false,
     });
   }
```

Comment:
instead of changing this, could you add a `MappingThree` that has the multi level path?

Reply:
I updated the existing mapping as I thought this test case to be a superset of the previous one. 
Will go ahead and create a new one instead now.

Reply:
This is true, but having the simpler case as a separate check will allow for finer-grained debugging if someone working in this module breaks it.